### ISSUE TITLE
Manipulating selected item in underlying select should update the state of the button

### ISSUE THREAD

**Issue by cammchugh** [dubbs/bootstrap-select-to-button#1]
When changing the selected value of the select list, the button state is not updated to reflect the new value of the select list. I might be missing something, but the documentation is currently a bit light, so apologies if this is already supported.

**Comment by dubbs** [comment]
You're right, I plan on adding update handlers for whenever the underlying select changes, eg:

``` js
$('select').prop('disabled', true);
$('select option:eq(0)').prop('selected', true);
```

**Comment by cammchugh** [comment]
Awesome, thanks.

**Comment by dubbs** [comment]
fixed by #2
